profiles/api/ping.py

```diff
@@ -13,4 +13,10 @@ def _ping() -> Response:
 
         return response
 
+    @blueprint.route('/error')
+    def _error() -> Response:
```

Comment:
Was this meant to be removed later?

Reply:
useful for testing purposes, could be enabled only on some environments? Although, it's useful even to test the production deployment of logging